shell/linux/Makefile

```diff
@@ -117,14 +117,17 @@ else ifneq (,$(findstring x64,$(platform)))
     endif
 
 # Generic 32 bit ARMhf (a.k.a. ARMv7h)
```

Comment:
If it's necessary to change "armv7h" to "armv7" as you're doing below, change it here too.